looprpc/client.proto

```diff
@@ -443,6 +449,12 @@ message QuoteResponse {
     On-chain cltv expiry delta
     */
     int32 cltv_delta = 5;
+
+    /**
+    The status of the quote. If this status is not 0 (QUOTE_OK) then an
+    information message should be shown to the user.
+    */
+    Status status = 6;
```

Comment:
I know I suggested this myself, but seeing this now I am not sure anymore. Suppose there would be another field that has a problem, how can that be communicated through `status`? It would need to be a flagged enum. Maybe a magic value for `miner_fee` wasn't that bad. Instead of `0`, we could also have picked `-1` to make it really clear. Although `0` miner fee also should never happen.

Reply:
Ok, switched back to magic number -1.